starter-code/css/main.css

```diff
@@ -1,8 +1,176 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:300,400&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  color: #1a1a1a;
+  font-family: "Roboto", sans-serif;
+}
 /*
 Colors:
 
 Text: 1A1A1A
-Green: #00B172
+Green: 
 White: #FFF
 
 */
+
+.header {
+  background-image: url("./../images/landing.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: 0px 0px;
+  height: 100vh;
+  position: relative;
+}
+
+.nav-bar {
+  background: #fff;
+  padding: 10px 20px;
+}
+
+.spotify-logo {
+  width: 10rem;
+}
+
+.nav-list {
+  width: 30%;
+  float: right;
+  margin: 16px 0;
+}
+
+.nav-list ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+.nav-list ul li {
+  list-style-type: none;
+  padding: 0 1rem;
+}
+
+.nav-list a {
+  text-decoration: none;
+  color: #1a1a1a;
+}
+
+h1 {
```

Comment:
For the titles of the header, you could also use the flex-box to center your content. But what you did, works too!